extensions/wikia/GlobalNavigation/styles/GlobalNavigation.scss

```diff
@@ -91,5 +92,142 @@
 	display: none;
 }
 
+.global-navigation-2016 {
+	.global-nav-search-input-wrapper .autocomplete-container,
+	.notifications-container .notifications-container,
```

Comment:
It's not the same class - it has different parent :) - it's just named same way as other one, but see below - we have 2 more with different names!